Content.Shared/Kitchen/MicrowaveMealRecipePrototype.cs

```diff
@@ -52,5 +52,21 @@ public FixedPoint2 IngredientCount()
             }
             return n;
         }
+
+        /// <summary>
+        ///     Sums the quantity of reagents in a recipe for sorting the recipe list.
+        ///     A fallback check if the IngredientCount is equal when sorting the
+        ///     recipe list.
+        /// </summary>
+        /// <returns></returns>
+        public FixedPoint2 ReagentQuantity()
```

Comment:
Please add a comment indicating that this method is floof-specific (either one comment before the block, or two comments surrounding it (section start/end)